ncm-filesystems/src/main/pan/components/filesystems/schema.pan

```diff
@@ -8,10 +8,14 @@ include 'quattor/schema';
 include 'quattor/blockdevices';
 include 'quattor/filesystems';
 
+@documentation{
+when manage_blockdevs is false, filesystems does same as fstab
```

Comment:
the whole `dangerous code isolated` is FUD unless you also don't use `ncm-symlink` and probably some other components that can easily do inrepairable damage. i tought that it was sufficient that the code was not in `fstab.pm`; i never understood it as `not in ncm-fstab`.
anyway, if this is the agreement, so be it.